py_gtfs_rt_ingestion/ingest.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import gzip
+import json
+import sys
+
+from pathlib import Path
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+from py_gtfs_rt_ingestion.helpers import get_datetime_from_header
+from py_gtfs_rt_ingestion.helpers import get_vehicle_schema
+
+DESCRIPTION = ""
+
+
+def parseArgs(args) -> dict:
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument(
+        '--fname',
+        dest='fname',
+        type=str,
+        required=True,
+        help='provide filename to ingest')
+
+    parsed_args = parser.parse_args(args)
+
+    return parsed_args
```

Comment:
Moved this into its own method. This way we can test our argparser all on its own and expand on it. It also means that the `run` method is taking in a filename rather than a list of sysvar args that need parsing.